scipy/spatial/distance.py

```diff
@@ -1421,10 +1481,6 @@ def dfun(u, v):
         elif mstr in ['minkowski', 'mi', 'm']:
             X = _convert_to_double(X)
             _distance_wrap.pdist_minkowski_wrap(X, dm, p)
-        elif mstr in ['wminkowski', 'wmi', 'wm', 'wpnorm']:
-            X = _convert_to_double(X)
-            w = _convert_to_double(np.asarray(w))
-            _distance_wrap.pdist_weighted_minkowski_wrap(X, dm, p, w)
```

Comment:
Weighted minkowski has never been correctly implemented--it's always been mathematically incorrect. Since there's not way to be backwards compatible anyway (unless we want to reintroduce the bug), I think it makes sense to just axe it.